TWLight/users/templates/users/collection_tile.html

```diff
@@ -9,83 +9,67 @@
 {% endcomment %}
 
 <div class="panel panel-default full-width">
-  <div class="panel-body top-border {% if each_authorization.about_to_expire %}expiry-warning{% endif %}">
-    {% if each_authorization.is_bundle %}
+  <div class="panel-body top-border {% if collection.authorization.about_to_expire %}expiry-warning{% endif %}">
```

Comment:
collection is the user-facing name of streams. Maybe `my_library_resource`? I realize the template is called collection tile, which is also not ideal.